Ethinylestradiol/Ethinylestradiol-Report.md

```diff
@@ -0,0 +1,419 @@
+| logD                            |                   | 3.63 - 3.9       | [DrugBank DB00977](#5-references) | Distribution coefficient                       |
+| fu<sup>+</sup>                  | %                 | 3                | [DrugBank DB00977](#5-references) | Fraction unbound in plasma                     |
+| CYP1A2 CL<sup>+</sup>           | µl/min/pmol       | 0.51             | [Ezuruike 2018](#5-references)    | Clearance by CYP1A2                            |
+| CYP2C8 CL<sup>+</sup>           | µl/min/pmol       | 0.13             | [Ezuruike 2018](#5-references)    | Clearance by CYP2C8                            |
+| CYP2C9 CL<sup>+</sup>           | µl/min/pmol       | 0.51             | [Ezuruike 2018](#5-references)    | Clearance by CYP2C9                            |
+| CYP3A4 CL<sup>+</sup>           | µl/min/pmol       | 0.5              | [Ezuruike 2018](#5-references)    | Clearance by CYP3A4                            |
+| Km UGT1A1<sup>+</sup>           | µmol/l            | 19.22            | [Ezuruike 2018](#5-references)    | UGT1A1 saturation constant                     |
+| Vmax UGT1A1<sup>+</sup>         | pmol/min/mg prot. | 408.5            | [Ezuruike 2018](#5-references)    | Maximal metabolization rate by UGT1A1          |
+| Renal Elimination<sup>+</sup>   | l/h               | 2.079            | [Stanczyk 2013](#5-references)    | Renal clearance                                |
+| Clint HLM<sup>+</sup>           | µL/min/mg prot.   | 118.83           | [Ezuruike 2018](#5-references)    | Intrinsic clearance in Human Liver Microsomes |
+| Ki CYP1A2                       | µmol/l            | 10.6             | [Karjalainen 2008](#5-references) | CYP1A2 inhibition constant                     |
+
+**Table 1:**<a name="Table 1"></a> Physico-chemical and *in-vitro* metabolization properties of ethinylestradiol extracted from literature. *<sup>+</sup>: Value used in final model*
+
+### 2.2.2 Clinical data
+
+A literature search was performed to collect available clinical data on ethinylestradiol ([Table 2](#Table 2)).
+
+| **Source**           | Route | **Dose [mg]/**  **Schedule \*** | **Pop.**     | **Sex** | **N** | **Form.** |
+| -------------------- | ------------------------------- | ------------ | ------- | --------------------------------- | --------------------------------- | --------------------------------- |
+| [Back 1981](#5-references)<sup>+</sup> | i.v.   | 0.03                            | HV       | F       | 5     | solution            |
+| [Back 1981](#5-references)<sup>+</sup>          | p.o.  | 0.03                            | HV       | F       | 5     | tablet              |
+| [Back 1979](#5-references)<sup>+</sup>          | i.v.  | 0.05                            | HV       | F       | 6     | solution            |
+| [Back 1979](#5-references)<sup>+</sup>          | p.o.  | 0.05                            | HV       | F       | 6     | NA                  |
```

Comment:
Maybe worth mentioning which formulation was used for Back 1979 po (NA in the table)?

Reply:
Issue created in ethinylestradio-repo